app/build.gradle

```
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        applicationId "com.sun_asterisk.foodies"
+        minSdkVersion 26
```

chơi đến 26 thì kiếm đâu ra user chú
```suggestion
        minSdkVersion 21
```

---

Dạ để em edit lại version